packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -1746,4 +1746,49 @@ void main() {
       areCreateAndDispose,
     );
   });
+
```

Comment:
Can you add back the test you had for the floating action button? That way we know we fixed both scenarios where this bug presented.

Reply:
i updated